### DISCUSSION TITLE
Raspberry Pi4 4Gb Ram | Full node Not Synced

### DISCUSSION THREAD

**Discussion by ziomik** [Chia-Network/chia-blockchain#5309] [Support]
Hi, Raspberry Pi with 4Gb ram, Samsung microSD 160/90 MB / sec, port 8444 correctly open. It has been in sync for two days but cannot reach the blockchain. I noticed many using the raspberry for the fullnode so I don't understand what it is that doesn't allow me to be synchronized
```
(venv) pi@raspberrypi:~/.chia/mainnet/log $ chia show -s
Current Blockchain Status: Full Node syncing to block 295280
Currently synced to block: 295223
Current Blockchain Status: **Not Synced**. Peak height: 295223
      Time: Mon May 17 2021 16:42:22 CEST                  Height:     295223
```

**Comment by kshyshkin** [comment]
Have long time this problems. Not solve it, but, if I close 8444, port, make rpi only for harvest, have sync wallet and node full. If I close port, make rpi as farmer, have not sync with 2... more hours delay all time. Sync goes but slow, and have lag in 2 or more hours. If make all as in wiki, have no sync at all....

**Reply by ziomik** [reply]
Does this compromise with harvest? I'm saying .. Do you know if to use the Rasp as harvest it is compulsory to have a fullnode sync?

**Reply by kshyshkin** [reply]
I think problem can be around low speed SD card operations or low processor. In my pc, with speed i7 and 12 treads, I see that load of processor about 20% when pc sync, farm and harvester.

**Reply by kshyshkin** [reply]
> Does this compromise with harvest? I'm saying .. Do you know if to use the Rasp as harvest it is compulsory to have a fullnode sync?

No, I think. I can turn on only harvester, and in logs see, that main pc send to rpi , and rpi reply ... but I keep sync, for future, many issues about sync, and decide keep sync on both rpi and pc

**Reply by ziomik** [reply]
> I think problem can be around low speed SD card operations or low processor. In my pc, with speed i7 and 12 treads, I see that load of processor about 20% when pc sync, farm and harvester.

Ok, this might make sense. But my doubt as to whether the block search must necessarily have a synchronized full node is not very clear. I notice on the LOG that he runs the plots were eligible for farming and often my plots are present even if I have 1 hour of desync.

**Reply by ziomik** [reply]
> > Does this compromise with harvest? I'm saying .. Do you know if to use the Rasp as harvest it is compulsory to have a fullnode sync?
> 
> No, I think. I can turn on only harvester, and in logs see, that main pc send to rpi , and rpi reply ... but I keep sync, for future, many issues about sync, and decide keep sync on both rpi and pc

ok i only saw your answer. I would like to have a certainty from some DEV. Thanks to you for the kind conversation

**Reply by kshyshkin** [reply]
Try like me, and will be have sync on both rpi and pc.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
